src/UI/Factory.php

```diff
@@ -336,4 +337,69 @@ public function panel();
+	 *     All Popovers contain a pointer pointing from the Popover to the Trigger of the Popover.
+	 *   effect: >
+	 *     Popovers are shown by clicking a Triggerer component such as a Button or Glyph.
+	 *     The position of the Popover is calculated automatically be default. However, it is possible to
+	 *     specify if the popover appears horizontal (left, right) or vertical (top, bottom) relative to
+	 *     its Triggerer component. Popovers disappear by clicking anywhere outside the Popover or by pressing
+	 *     the ESC key.
+	 * rivals: >
+	 *   Modals: >
+	 *     Modals hide all other content while Popovers do not prevent interaction with other parts
+	 *     of the current context.
+	 * context:
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *        Popovers MUST NOT contain horizontal scrollbars. The renderer of the content component is responsible
+	 *        that the content does not exceed the max width from the Popover.
+	 *     2: >
+	 *        Popovers MAY contain vertical scrollbars. The renderer of the content component is responsible to
+	 *        set the max height and showing the vertical scrollbars.
+	 *     3: >
+	 *        If Popovers are used to present secondary information of an object, they SHOULD display a title
+	 *        representing the object.
+	 *   interaction:
+	 *     1: >
+	 *        A Popover MUST only be displayed if the Trigger component is clicked.
+	 *        This behaviour is different from Tooltips that appear on hovering.
+	 *   style:
+	 *     1: Popovers MUST always relate to the Trigger component by a little pointer.
+	 *   accessibility:
+	 *     1: >
+	 *        There MUST be a way to open the Popover by only using the keyboard. Exceptions are acceptable
+	 *        for inputs that can be entered by only using the keyboard without activating the Popover.
+	 *        In this case the input element MUST indicate the required format by setting its placeholder accordingly.
+	 *     2: >
+	 *        The focus MUST be inside the Popover, once it is open if it contains at least one interactive item.
+	 *        Otherwise the focus MUST remain on the Triggerer component.
+	 *     3: >
+	 *        The focus MUST NOT leave the Popover for as long as it is open.
+	 *     4: >
+	 *        There MUST be a way to reach every control in the Popover by only using the keyboard.
+	 *        Exception are acceptable for controls that are only there for convenience (e.g. elements used to
+	 *        reduce the number of clicks to achieve a certain input such as the switch from the month-view
+	 *        to the year-view in the Date/Time picker).
+	 *     5: >
+	 *        The Popover MUST be closable by pressing the ESC key.
+	 *     6: >
+	 *        Once the Popover is closed, the focus MUST return to the element triggering the opening of the Popover
+	 *        or the element being clicked if the Popover was closed on click.
+	 * ---
+	 * @param Component|Component[] $content
```

Comment:
I think we need to find some middle ground here. How about introducing a "standard" popover accepting any content for the moment and one "listing" only accepting a set of items. See https://github.com/ILIAS-eLearning/ILIAS/pull/543/files#diff-50f8f3ede5cc40ae3097cc424fb6a839. We can also think of leaving out the "standard" for the moment and only introduce the "listing" since this is the one we have a very concrete use case for 5.3 (background tasks). @klees and @wanze what do you think?